io/zenoh-transport/src/common/pipeline.rs

```diff
@@ -655,326 +682,343 @@ impl TransmissionPipelineConsumer {
+//         let count = Arc::new(AtomicUsize::new(0));
+//         let size = Arc::new(AtomicUsize::new(0));
+
+//         let c_size = size.clone();
+//         task::spawn(async move {
+//             loop {
+//                 let payload_sizes: [usize; 16] = [
+//                     8, 16, 32, 64, 128, 256, 512, 1_024, 2_048, 4_096, 8_192, 16_384, 32_768,
+//                     65_536, 262_144, 1_048_576,
+//                 ];
+//                 for size in payload_sizes.iter() {
+//                     c_size.store(*size, Ordering::Release);
+
+//                     // Send reliable messages
+//                     let key = "pipeline/thr".into();
+//                     let payload = ZBuf::from(vec![0_u8; *size]);
+//                     let channel = Channel {
+//                         priority: Priority::Control,
+//                         reliability: Reliability::Reliable,
+//                     };
+//                     let congestion_control = CongestionControl::Block;
+//                     let data_info = None;
+//                     let routing_context = None;
+//                     let reply_context = None;
+//                     let attachment = None;
+
+//                     let message = ZenohMessage::make_data(
+//                         key,
+//                         payload,
+//                         channel,
+//                         congestion_control,
+//                         data_info,
+//                         routing_context,
+//                         reply_context,
+//                         attachment,
+//                     );
+
+//                     let duration = Duration::from_millis(5_500);
+//                     let start = Instant::now();
+//                     while start.elapsed() < duration {
+//                         producer.push_zenoh_message(message.clone());
+//                     }
+//                 }
+//             }
+//         });
+
+//         let c_count = count.clone();
+//         task::spawn(async move {
+//             loop {
+//                 let (batch, priority) = consumer.pull().await.unwrap();
+//                 c_count.fetch_add(batch.len() as usize, Ordering::AcqRel);
+//                 consumer.refill(batch, priority);
+//             }
+//         });
+
+//         task::block_on(async {
+//             let mut prev_size: usize = usize::MAX;
+//             loop {
+//                 let received = count.swap(0, Ordering::AcqRel);
+//                 let current: usize = size.load(Ordering::Acquire);
+//                 if current == prev_size {
+//                     let thr = (8.0 * received as f64) / 1_000_000_000.0;
+//                     println!("{} bytes: {:.6} Gbps", current, 2.0 * thr);
+//                 }
+//                 prev_size = current;
+//                 task::sleep(Duration::from_millis(500)).await;
+//             }
+//         });
+//     }
+// }
```

Comment:
all tests are commented out, just checking that you didn't simply forget to re-enable them after working on the file

Reply:
I forgot to re-enable them. Fixed.